modules/ctrl_dbus/CMakeLists.txt

```diff
@@ -0,0 +1,47 @@
+project(ctrl_dbus)
+
+set(SRCS ctrl_dbus.c baresipbus.c)
+
+find_package(PkgConfig REQUIRED)
+
+pkg_check_modules(GLIB REQUIRED glib-2.0>=0.29.1)
+pkg_check_modules(GIO  REQUIRED gio-unix-2.0>=0.29.1)
```

Comment:
These checks should be moved to a separat FindGLIB.cmake module in the `cmake` folder, so the main CMakeLists.txt includes `ctrl_dbus` only if requirements are met (like AAC).

Reply:
okay, thanks!

Reply:
I added a minimalist `FindGLIB.cmake`.

Do we need the other find methods like in `FindAAC.cmake` and the very first `FindOPUS.cmake` ? This seems to be a manual method. Modul `PkgConfig` should do all this alreaydy.

Reply:
Answer myself:
The cmake manual suggest to add more checks. I can't understand why this is needed.
https://cmake.org/cmake/help/latest/manual/cmake-developer.7.html

Anyway, added these checks like in the other cmake/Find... modules.